src/value.h

```diff
@@ -84,6 +85,10 @@ typedef struct SIValue {
 		void *ptrval;
 		struct Pair *map;
 		struct SIValue *array;
+		struct {
+			float latitude;   // 32 bit
+			float longitude;  // 32 bit
```

Comment:
I understand the desire for a struct size of 8, but are floats sufficiently precise for lat/lon values? I saw a lot of floating point error with some minor testing.

If this is a significant problem, we should switch to a heap-allocated struct.